checkov/common/bridgecrew/vulnerability_scanning/image_scanner.py

```diff
@@ -109,5 +114,29 @@ def scan(self, docker_image_id: str, dockerfile_path: str, skip_extract_image_na
             logging.error("Failed to scan docker image", exc_info=True)
             return 1
 
+    def get_scan_results_from_cache(self, image_id: str) -> Dict[str, Any]:
+        headers = merge_dicts(
+            get_default_get_headers(bc_integration.bc_source, bc_integration.bc_source_version),
+            {"Authorization": bc_integration.get_auth_token()},
+        )
+        response = requests.request(
+            "GET", f"{bc_integration.api_url}/api/v1/vulnerabilities/scan-results/{image_id}", headers=headers
+        )
+
+        response_json = response.json()
+        output_type = response_json["outputType"]
+
+        if output_type == "Error":
+            logging.error(response_json["outputData"])
+            return {}
+
+        elif output_type == "Result":
+            return json.loads(
+                decompress_file_gzip_base64(
+                    response.json()["outputData"]
+                )
+            )
+        return {}
```

Comment:
i suggest one of the next here:
1. writing a comment here for describing that output_type == "Empty" in case it reaches to line 139, 
2.  adding third 'elif' (output_type == "Empty") statement and another else
in the else segment, just raise an error, so we are monitored in case there is new output_type which are not known here. 
basically, it is up to you
@nimrodkor what I suggest as option 2 is good practice or too much according to your opinion

Reply:
Would log an info log in this case, something like `Got an empty result for file {file_name}`.
1 - doesn't help the average user
2 - we can't monitor the cli runs on user computers... So we'll have to let them know, gently, so they can reach out. raising an error will crash the scan, and we don't want that if we did something wrong...